core/geometry/src/geometry3d/Point3dVector3d.ts

```diff
@@ -166,6 +169,14 @@ export class XYZ implements XYAndZ {
       && Geometry.isSameCoordinate(this.y, y, tol)
       && Geometry.isSameCoordinate(this.z, z, tol);
   }
+  /** Return true if this and {other + vector*scale} have equal x,y,z parts within Geometry.smallMetricDistance.
+   * * this method is useful in testing "point on ray" without explicitly constructing the projection point
+  */
```

Comment:
You have only added 2 new functions to Point3dVector3d.ts (i.e., isAlmostEqualPointPlusScaledVector and createUnitCrossProductToPoints)
Rest of the changes made to this file has (I suppose accidently) reverted the changes I made to this class. Please fix them.

Reply:
I'm glad you found that clearly . ..